### PR TITLE
Initial sample

### PR DESCRIPTION
**Pull request by derisen**
## Purpose

* This PR adds a Node.js console (cli) app that is calling MS Graph API using application permissions with MSAL Node.
* This sample is meant to be used as a **quickstart** sample.

## Does this introduce a breaking change?

```
[ ] Yes
[ x ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ x ] Other... Please describe: **new sample**
```

## How to Test
*  Get the code

```
git clone https://github.com/Azure-Samples/ms-identity-javascript-nodejs-console.git
cd ms-identity-javascript-nodejs-console
git checkout derisen
npm install
```

cc @hectormmg

### CODE REVIEW

**Review by hectormmg — APPROVED** [review]
LGTM!

File: README.md [hunk 1]
```diff
@@ -1,57 +1,75 @@
...
+* [Node.js](https://nodejs.org/en/) must be installed to run this sample.
+* [Visual Studio Code](https://code.visualstudio.com/download) is recommended for running and editing this sample.
 
-(ideally very short)
+### Setup
 
-- npm install [package name]
-- mvn install
-- ...
+1. [Register a new application](https://docs.microsoft.com/azure/active-directory/develop/scenario-daemon-app-registration) in the [Azure Portal](https://portal.azure.com). Ensure that you have obtained **admin consent** for API permissions.
```
Comment (idg-sam) [inline README.md]:
```suggestion
1. Register [a Daemon app that calls web APIs](https://docs.microsoft.com/azure/active-directory/develop/scenario-daemon-app-registration) in the [Azure Portal](https://portal.azure.com).
1. In the app's registration screen, select the API permissions blade in the left to open the page where we add access to the APIs that your application needs.
    - Select the Add a permission button and then,
    - Ensure that the Microsoft APIs tab is selected.
    - In the Commonly used Microsoft APIs section, select Microsoft Graph
    - Select Application Permissions. 
    - In this section, select the `User.Read.All` from the list. Use the search box if necessary.
    - Select the Add permissions button at the bottom.
    - Be sure to click the checkmark to **Grant admin consent**.
```

**Review by idg-sam — CHANGES_REQUESTED** [review]
Works great, I modified app registration steps in github suggestions so the user can register their daemon app and the relevant API permission.

File: README.md (hunk 1 above)
Comment (kalyankrishna1) [reply README.md]: +1, its "Application Permissions"

---
In reply to: [560588228](https://github.com/Azure-Samples/ms-identity-javascript-nodejs-console/pull/1#discussion_r560588228) [](ancestors = 560588228)

**Review by kalyankrishna1 — CHANGES_REQUESTED** [review]
:clock1:

**Review by samuelkubai — APPROVED** [review]
LGTM, went through the samples and works great.

**Pull request merged**